design/ceph/ceph-exporter.md

```diff
@@ -0,0 +1,81 @@
+# Ceph exporter
+
+### Goals
+
+Use Ceph exporter as the new source of metrics based on performance counters coming from every Ceph daemon.
+
+## Use case
+
+Ceph exporter was created to avoid performance problems at scale obtaining metrics in clusters with high number of daemons (mainly OSDs).
+The Prometheus manager module was not enough performant to obtain the performance counters from all the daemons along the whole Ceph cluster, so the solution for avoid this problem in big clusters is to have a new daemon collecting all the performance counters from all the daemons in each host, and expose them as standard prometheus metrics at /metrics endpoint for prometheus to scrape each of those endpoints.
+
+### Why do we need this daemon in Rook
+
+With the Ceph exporter, the source of metrics changes in the Ceph clusters.
+There will be two main sources of metrics:
+- Prometheus manager module: It is responsible for exposing all metrics other than ceph daemons performance counters.
+- Ceph exporter: It is responsible for exposing only ceph daemons performance counters as prometheus metrics.
+
+Ceph exporter will be deployed in Rook when the `monitoring.enabled: true` setting is applied in the CephCluster CR and the Ceph version later than 17.2.5.
```

Comment:
If it's not merged yet, how about we include a link to the ceph change so we can track it here
```suggestion
Ceph exporter will be deployed in Rook when the `monitoring.enabled: true` setting is applied in the CephCluster CR and the Ceph version planned starting with 17.2.6. The Ceph change is PR <link>
```

Reply:
PR is merged and has been backported to quincy as well. It''s just I wasn't sure which next version to mention, so I wrote later than 17.2.5

Reply:
Ok sounds good if it's merged and backported to quincy